pkg/annotations/annotations.go

```diff
@@ -44,6 +44,35 @@ const (
 
 	// CPUFreqGovernorAnnotation sets the cpufreq governor for CPUs used by the container
 	CPUFreqGovernorAnnotation = "cpu-freq-governor.crio.io"
+
```

Comment:
I feel like we should create a new file for these

Reply:
done